bin/telemetry_sim_LSSTCam

```diff
@@ -23,8 +23,17 @@ def cmdline():
     parser.add_argument("--rotpa", action="store", default=0.0, type=float,
                         help="The ROTPA for the visit")
 
+    parser.add_argument("--filterband", action="store", default='r',
+                        help="The filter band name (i.e. r)")
+    parser.add_argument("--filtername", action="store", default='U-001',
+                        help="Name of the physical filter (i.e. U-001)")
+    parser.add_argument("--filterslot", action="store", default=0,
+                        help="Filter slot (1-indexed)")
+    parser.add_argument("--filterpos", action="store", default=666.0,
+                        help="The filter measured position of slide (i.e.: 666.0)")
+
     # Control imageName
-    parser.add_argument("--camcode", action="store", default='MT',
+    parser.add_argument("--camcode", action="store", default='CC',
```

Comment:
The original line is correct. It should be MT.

Reply:
Fixed!

Reply:
Fixed